lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -93,11 +93,14 @@ defmodule ExDoc.Formatter.HTML.Autolink do
           entry <- [doc.id | doc.defaults],
           do: prefix <> module.id <> "." <> entry
 
+    lib_dirs = extra_lib_dirs ++ elixir_lib_dirs() ++ erlang_lib_dirs()
+    lib_dirs = Enum.map(lib_dirs, fn {path, url} -> {Path.expand(path), url} end)
```

Comment:
I would prefer if we did not do this and the paths were already given expanded. Would that be possible?

Reply:
@josevalim agreed. I just realised I was expanding elixir paths twice so doing it just in `elixir_lib_dir/1` now.